api/tests/core/criteria/test_api.py

```diff
@@ -0,0 +1,101 @@
+import pytest
+
+from pcapi.core.criteria import api as criteria_api
+from pcapi.core.offers import factories as offers_factories
+
+
+pytestmark = pytest.mark.usefixtures("db_session")
+
+
+def test_offers_update():
+    """
+    Test that a bulk criteria update creates the expected OfferCriterion
+    objects.
+    """
+    offers = offers_factories.OfferFactory.create_batch(3)
+    criteria = offers_factories.CriterionFactory.create_batch(3)
+
+    # Add criteria to offers:
+    #   * first offer has all criteria,
+    #   * second all expect the first,
```

Comment:
typo sur expect/except ?

```suggestion
    #   * second venue: 2nd and 3rd criteria only,
```